langchain/llms/anthropic.py

```diff
@@ -102,7 +102,7 @@ def _wrap_prompt(self, prompt: str) -> str:
         if prompt.startswith(self.HUMAN_PROMPT):
             return prompt  # Already wrapped.
         else:
-            return f"{self.HUMAN_PROMPT} {prompt}{self.AI_PROMPT} Sure, here you go:\n"
```

Comment:
I think you want to keep this, for compatibility with existing chains. Otherwise in response to e.g. a summarization prompt Claude will respond in a verbose way like

"Sure, here is a summary of the above document: "

which the downstream chain may not be expecting